web/src/components/ToggleButton.tsx

```diff
@@ -79,16 +80,16 @@ export default function ToggleButton({
                 role="button"
                 tabIndex={0}
                 className={twMerge(
-                  'ml-2 inline-flex h-[29px] w-[29px] select-none justify-center self-center rounded-full bg-white shadow dark:border dark:border-gray-500/80 dark:bg-gray-600/50',
+                  'inline-flex select-none items-center justify-center',
                   isToolTipOpen && 'pointer-events-none'
                 )}
               >
-                <div className="self-center text-xs font-bold">i</div>
+                <Info className="text-gray-500" />
```

Comment:
I think the icon should be a bit bigger
```suggestion
                <Info className="text-gray-500" size={30} />
```

Reply:
I considered it but then it felt too big compared to the other info icons (the one in the zone header in particular) but we can try bumping it a bit and see if you wish?

Reply:
Now it looks like this:
<img width="319" alt="image" src="https://github.com/user-attachments/assets/d6812b84-f754-4df6-a60e-dad6407598a8">


Not a huge fan of it as now the icons look miss aligned and too big compared to the rest?

Reply:
I prefer it as it is right now as it aligns with the outer circle. If you want to we can probably get alex take on it tomorrow ?

Reply:
Sounds good, there is no hurry here so it's better we take our time with it.

Reply:
I just saw Alex had made a prototype design for this so I'll go ahead and implement that: 
<img width="552" alt="image" src="https://github.com/user-attachments/assets/09d8c396-eaa1-402c-b21e-f7b5c0beb38c">

Should fix both our concerns I think.